src/main/java/uk/gov/mca/beacons/api/dto/LegacyBeaconDTO.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.mca.beacons.api.dto;
+
+import static uk.gov.mca.beacons.api.dto.LegacyBeaconDTO.Attributes;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+public class LegacyBeaconDTO extends DomainDTO<Attributes> {
+
+  private String type = "legacyBeacon";
+
+  public String getType() {
+    return type;
+  }
+
+  @Getter
+  @Setter
+  @Builder
+  @NoArgsConstructor
+  @AllArgsConstructor
+  public static class Attributes {
+
+    private Map<String, Object> beacon;
+    private List<Map<String, Object>> owners;
```

Comment:
Split this out into:
- Owner
- Secondary owners
And have the ETL process send across the main owner and secondary owners